native-code/ios/index.md

```diff
@@ -45,69 +52,54 @@ to update the code in your checkout.
 [GN][5] to generate build files for iOS certain variables need to be set.
 Those variables can be edited for the various build configurations as needed.
 
-The component build is the default for Debug builds, which are also enabled by
-default unless `is_debug=false` is specified. iOS needs static builds, which is
-why `is_component_build=false` is specified for all the examples above.
-
-
-#### Targeting iOS Devices
-
-The different output directories can be replaced with any directory of your own
-choice. When running GN; make sure your current working directory is src/ of
-your workspace, then run:
-
-~~~~~ bash
-gn gen out/Debug-device-arm32 --args='target_os="ios" target_cpu="arm" is_component_build=false'
-~~~~~
-
-#### Targeting 64-bit iOS devices
-
-~~~~~ bash
-gn gen out/Debug-device-arm64 --args='target_os="ios" target_cpu="arm64" is_component_build=false'
-~~~~~
-
-**NOTICE:** To build for devices, you must have a signing identity configured
-for your Xcode installation. You can check this by running:
+The variables you should care about are the following:
+
+* `target_os`:
+  - To build for iOS this should be set as `target_os="ios"` in your `gn args`.
+  The default is whatever OS you are building on.
+* `target_cpu`:
+  - For builds targeting iOS devices, this should be set to either `"arm"` or
+  `"arm64"`, depending on the architecture of the device. For builds to run in
+  the simulator, this should be set to `"x64"`.
+* `is_component_build`:
+  - Component builds don't take as long to link, but have runtime performance
+  implications. They are not supported on iOS, so this should always be set
+  to `false`.
+* `is_debug`:
+  - Debug builds are the default. When building for release, specify `false`.
```

Comment:
I agree, but it's a little different and not part of `gn args`. I think it should be more prominent, but does not belong in this list.